src/components/github-button.tsx

```diff
@@ -30,11 +30,14 @@ const GitHubButton = () => {
       target='_blank'
       rel='noopener noreferrer'
       className='inline-flex items-center rounded-lg px-4 py-2 text-black shadow transition duration-200'
-      style={{ background: '#d9d9d980' }}
     >
-      <Image src={githubIcon} alt='Github star icon' className='h-5 w-5' />
-      <span className='ml-2 font-medium'>Contribute</span>
-      {starCount !== null && <span className='ml-2'>⭐ {starCount}</span>}
+      <Image src={githubIcon} alt='Github star icon' className='mr-2 h-5 w-5' />
+      <span className='hidden text-sm font-medium sm:inline'>
+        {stars !== null ? `Contribute ${stars} ★` : 'Loading...'}
```

Comment:
Please change this to the Star Emoji: ⭐️
and I would be happy to merge the PR
Cheers :)